layouts/New/Blog.tsx

```diff
@@ -41,9 +44,17 @@ const BlogLayout: FC = async () => {
 
       <div className={styles.blogLayout}>
         <main>
-          <h1>{t('layouts.blog.title')}</h1>
-
-          <p>{t('layouts.blog.subtitle')}</p>
+          <header>
+            <h1>
+              {t('layouts.blog.title')}
+              <Link
+                href={`/feed/${siteConfig.rssFeeds.find(item => item.blogCategory === blogData.category)?.file ?? 'blog.xml'}`}
```

Comment:
This logic seems broken.

Reply:
why do you mean ? I have manually test on local and it's work.

Reply:
It doesn't work on the preview branch. The preview branch is the closest to a production environment.

Reply:
"I have manually test on local and it's work." 

https://www.reddit.com/r/ProgrammerHumor/comments/70we66/it_works_on_my_machine/

Reply:
FOUND it's because type annotation isn't right.

Reply:
Looks broken on this category https://nodejs-org-git-featrss-visible-link-openjs.vercel.app/en/blog/announcements, shouldn't it show `blog.xml`?

Reply:
> shouldn't it show blog.xml?

yeah, I think